spec/unit/network/http_pool_spec.rb

```diff
@@ -49,7 +50,7 @@ def initialize(host, port, options = {})
       expect(Puppet::Network::HttpPool.http_ssl_instance("me", 54321)).to be_use_ssl
     end
 
-    context "when calling 'connection'" do
+    context "when calling 'connection'", :unless => Puppet::Util::Platform.jruby? do
```

Comment:
Related to comment above, I'd expect to run these tests on JRuby

Reply:
I've removed the conditional from the overall context and moved it down to the examples that test the error/warnings.